migmutatingwebhook/migmutatingwebhook.go

```diff
@@ -0,0 +1,122 @@
+package main
+
+import (
+	"crypto/rsa"
+	"crypto/tls"
+	"crypto/x509"
+	"encoding/pem"
+	"log"
+	"migmutatingwebhook/pkg/certhandler"
+	"migmutatingwebhook/pkg/secretmanager"
+	webhookserver "migmutatingwebhook/pkg/server"
+	"net/http"
+)
+
+var (
+	serverTLSCert    *tls.Config
+	clientTLSCert    *tls.Config
+	caPEM            []byte
+	rawLeafCert      []byte
+	bytePrivateKey   []byte
+	stringLeafCert   string
+	stringCACert     string
+	stringPrivateKey string
+	err              error
+)
+
+func main() {
+
+	log.Print("Initializing certificates")
+
+	certSecretExists := secretmanager.GetSecret()
+
+	if certSecretExists == true {
+
+		log.Print("Secret was found. Starting server using existing tls configuration")
+
+		migmutatingwebhookTLSSecret := secretmanager.GetCertSecretData()
+
+		for key, value := range migmutatingwebhookTLSSecret.Data {
+
+			if key == "ca.crt" {
+
+				caPEM = value
```

Comment:
```suggestion
			if key == "ca.crt" {
				caPEM = value
```

same at other places